File: SDKs/nordigen.com/1.7/typescript/src/sdk/models/shared/requisitionlinkparams.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// RequisitionLinkParams
/** 
 * RequisitionLinkParamsSerializer.
**/
export class RequisitionLinkParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=aspsp_id, form, name=aspsp_id;, multipart_form, name=aspsp_id" })
  aspspId: string;
}

File: SDKs/nordigen.com/1.7/python/src/sdk/models/shared/requisitionlinkparams.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class RequisitionLinkParams:
    r"""RequisitionLinkParams
    RequisitionLinkParamsSerializer.
    """
    
    aspsp_id: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('aspsp_id') }, 'form': { 'field_name': 'aspsp_id' }, 'multipart_form': { 'field_name': 'aspsp_id' }})
    
